apps/main/src/loan/components/PageLoanCreate/Page.tsx

```diff
@@ -184,8 +176,8 @@ const Page = () => {
               isReady={isReady}
               isLeverage={isLeverage}
               loanExists={loanExists}
-              llamma={llamma}
-              llammaId={llammaId}
+              llamma={market ?? null}
```

Comment:
Markets can be undefined if they're not yet loaded. The old components however rely on `null` for whatever reason. Didn't want to touch those to keep the PR small.